worker/mediasoup-worker.gyp

```diff
@@ -271,6 +271,13 @@
         'ldflags': [ '-Wl,--export-dynamic' ]
       }],
 
+      [ 'OS == "win"', {
+        'sources': [
+          'include/opt/getopt.h',
+          'opt/getopt.cpp'
+        ]
+      }],
```

Comment:
As told above, I'd prefer that  `getopt` is under `worker/deps` as many other dependencies.

Reply:
already move getopt into deps.